src/utils/mediasource-helper.ts

```diff
@@ -15,3 +15,7 @@ export function getMediaSource(
     ((self as any).WebKitMediaSource as typeof MediaSource)
   );
 }
+
+export function isManagedMediaSource(source: typeof MediaSource | undefined) {
+  return source === (self as any).ManagedMediaSource;
```

Comment:
`self` may be undefined in some environments.

```suggestion
  return typeof self !== 'undefined' &&
    source === (self as any).ManagedMediaSource;
```

Reply:
I like this, thank you